projectX/main.py

```diff
@@ -3,23 +3,40 @@
 import argparse
 
 import Sparkassenparser
-from TagConfig import TagConfig
+from TagConfig import TagConfig, RegexIdentifier
 from Tagger import Tagger
 
 
 parser = argparse.ArgumentParser("main script")
-parser.add_argument('-c', '--csv', required=True)
-parser.add_argument('-tc', '--tagconfig', required=True)
+parser.add_argument('-c', '--csv')
+parser.add_argument('-tc', '--tagconfig')
 
 args = parser.parse_args()
 
 csv_path = args.csv
 tagconfig_path = args.tagconfig
 
-def main():
+def main(csv_p, tc_p):
+    tc = TagConfig()
+    tc.add_category('Grundausgaben')
```

Comment:
Brauchen wir die config hier? "Grundausgaben", "Vattenfall", etc..

Reply:
Das ist nur ein Beispiel wie man die config neu anlegt. Später kann man so zu neuen Buchungen die config um Kategorie (+ Buchungstyp + Identifikation) erweitern.

Als Beispiel habe ich eine Kategorie mit zwei Buchungstypen erstellt die dann genutzt werden können um je eine Abbuchung in der example.csv zu 'matchen'

Reply:
Später definitiv. Das ist nur als Test und zur Demonstration wie die config funktioniert. Später würde vermutlich die main nur aus dem cli/menu bestehen.

Reply:
Können wir gerne so machen, wir bauen damit nur Code-Schulden auf, mit diesen Verweisen auf "später" die du bringst.
Gerade ist dieser Code nicht clean, teile der Config liegen hardcoded in main() um zu erklären wie der Code funktioniert.